runtime/kernel/operator_registry.cpp

```diff
@@ -88,24 +95,33 @@ Error register_kernels_internal(const Span<const Kernel> kernels) {
           kernel.kernel_key_ == k.kernel_key_) {
         ET_LOG(Error, "Re-registering %s, from %s", k.name_, lib_name);
         ET_LOG_KERNEL_KEY(k.kernel_key_);
-        return Error::RegistrationAlreadyRegistered;
+        err = Error::RegistrationAlreadyRegistered;
+        continue;
       }
     }
+
     registered_kernels[num_registered_kernels++] = kernel;
   }
-  ET_LOG(
-      Debug,
-      "Successfully registered all kernels from shared library: %s",
-      lib_name);
 
-  return Error::Ok;
+  if (errorHandler != nullptr) {
+    err =  errorHandler(err);
```

Comment:
This makes me a bit nervous. Why do we want to consume an `Error` and translate it to another `Error`? In practice, are you going to add product specific `Error` like `LoggingFailure` etc?